Fedcloud documentation

enolfc opened the pull request:
<!--
A good PR should describe what benefit this brings to the repository.
Ideally, there is an existing issue which the PR address.
-->

# Summary 

Migrate fedcloud documentation
<!-- Describe in plain English what this PR does -->

----
<!-- Add the related issue here, e.g. #6 -->
**Related issue :**

gwarf commented:
We can certainly decide to ignore MD026: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md026---trailing-punctuation-in-heading

In the `.travis.yml`
```
-r ~MD026
```
Code style should probably be aligned
https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md046---code-block-style

And for `MD024 Multiple headers with the same content` if it makes sense to keep similar headers we can force the merge.

enolfc commented:
> We can certainly decide to ignore MD026: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md026---trailing-punctuation-in-heading
> 
> In the `.travis.yml`
> 
> ```
> -r ~MD026
> ```

I added specific style to deal with this, allowing ? in headers, which is the main issue with FAQs.

> 
> Code style should probably be aligned
> https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md046---code-block-style
> 
> And for `MD024 Multiple headers with the same content` if it makes sense to keep similar headers we can force the merge.

I removed this one, as I believe it makes sense to have this kind of similar headers
